plugins/janus_streaming.c

```diff
@@ -691,8 +692,8 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 				}
 				if(doaudio &&
 						(aport == NULL || aport->value == NULL ||
-						acodec == NULL || acodec->value == NULL ||
-						artpmap == NULL || artpmap->value == NULL)) {
+						atoi(aport->value) == 0 || acodec == NULL ||
```

Comment:
Can you move the `atoi(aport->value) == 0` part a couple lines above, so that it's next to the `aport->value == NULL` check? And bring the two `acodec` checks on the same line again? It's cleaner from a code perspective.